types/types.go

```diff
@@ -6,9 +6,62 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+// BaseEntity describes a generic MongoDB row
 type BaseEntity struct {
 	ID        bson.Binary `bson:"_id,omitempty" json:"id"`
 	Name      string      `bson:"name" json:"name"`
 	CreatedAt time.Time   `bson:"created_at" json:"created_at"`
 	UpdatedAt time.Time   `bson:"updated_at" json:"updated_at"`
 }
+
+// MonsterEntity describes the general stats of a Monster
+type MonsterEntity struct {
```

Comment:
why would I need the bson if the payload is going to be retrieved from the API as JSON? What is the purpose of having both BSON and JSON mappings?